internal/server/openapi/api_model_registry_service_service.go

```diff
@@ -405,50 +339,41 @@ func (s *ModelRegistryServiceAPIService) GetModelVersionArtifacts(ctx context.Co
 	// TODO externalID unused
 	listOpts, err := apiutils.BuildListOption(pageSize, orderBy, sortOrder, nextPageToken)
 	if err != nil {
-		return Response(500, model.Error{Message: err.Error()}), nil
+		return Response(http.StatusInternalServerError, model.Error{Message: err.Error()}), nil
```

Comment:
the only way `apiutils.BuildListOption` could err is user supplying a wrong content for pageSize,
so I would suggest this fn (`apiutils.BuildListOption`) should follow the same pattern,
i.e.: 
- make it return a semantic error,
- and use the conversion function. 

wdyt?
let me know if you want to do it as part of this PR, or in a following one (in which case on notification I'll proceed to merge this now)

Reply:
will update on this one, then. Thanks for the heads up!